Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -96,7 +95,7 @@ void GeneralConfigPane::LoadGUIValues()
 	m_idle_skip_checkbox->SetValue(startup_params.bSkipIdle);
 	m_cheats_checkbox->SetValue(startup_params.bEnableCheats);
 	m_force_ntscj_checkbox->SetValue(startup_params.bForceNTSCJ);
-	m_frame_limit_choice->SetSelection(SConfig::GetInstance().m_Framelimit);
+	m_throttler_choice->SetSelection(std::lround(SConfig::GetInstance().m_EmulationSpeed * 10.0f));
```

Comment:
What happens if the `SetSelection` index is out of range? Does wx do something like picking the first option or does it do something bad?